examples/array-map-symmetry/array_map_symmetry.bal

```diff
@@ -0,0 +1,40 @@
+import ballerina/io;
+
+public function main() {
+    // List constructors are used to construct list values.
+    string[] list = ["foo", "bar"];
+
+    // Mapping constructors are used to construct map values.
+    map<int> 'map = {x: 1, y: 2};
```

Comment:
If the list is `list` this can be `mapping`. Even if not, let's use some other name which will allow us to avoid using a quoted identifier.